subprojects/packagefiles/lmdb/tests/meson.build

```diff
@@ -0,0 +1,16 @@
+tests = [
+  'mtest',
+  'mtest2',
+  'mtest3',
+  'mtest4',
+  'mtest5'
+]
+
+rm = find_program('rm')
+mkdir = find_program('mkdir')
+
+foreach t : tests
+  exe = executable(t, '../libraries/liblmdb/@0@.c'.format(t),
+    dependencies: [lmdb_dep])
+  test(t, exe)
```

Comment:
The tests need a `testdb` directory created before they are run, so the flow goes like `mkdir testdb; ./test; rm -rf testdb` not able to find how to achieve this in meson

Reply:
The test can be something like (untested) `'python', '-c', import sys, os, subprocess; os.makedirs("testdb", exist_ok=True); subprocess.check_call([sys.argv[1], "testdb")', exe`

Reply:
Did it with `sh` since it's easier to do stuff. Also `rm -rf` is run each time since lmdb tests expect a clean database.